### PR TITLE
feat: mg experiment removed from lend menu

### PR DESCRIPTION
**Pull request by roger-in-kiva**
- clean up experiment in lend dropdown for monthly good

### CODE REVIEW

File: src/components/WwwFrame/LendMenu/LendListMenu.vue [hunk 1]
```diff
@@ -4,12 +4,9 @@
 			v-if="showMGUpsellLink"
 			to="/monthlygood"
 			@click.native="trackMgLinkClick"
-			:class="{ 'hover:tw-no-underline': newMgEntrypoint}"
 		>
-			<!-- NEW MG ENTRYPOINT CORE-641 -->
-			<monthly-good-entrypoint v-if="newMgEntrypoint" />
```
Comment (dyersituations) [inline src/components/WwwFrame/LendMenu/LendListMenu.vue:10]: There's still a couple instances of `newMgEntrypoint` than can be removed from this component.

**Review by dyersituations — APPROVED** [review]

**Comment by mcstover** [comment]
@roger-in-kiva @dyersituations Can this also be cleaned up in cms-page-server?

**Comment by dyersituations** [comment]
> @roger-in-kiva @dyersituations Can this also be cleaned up in cms-page-server?

Ah yup, good call. @roger-in-kiva please make the same changes in that repo!

**Comment by roger-in-kiva** [comment]
> > @roger-in-kiva @dyersituations Can this also be cleaned up in cms-page-server?
> 
> Ah yup, good call. @roger-in-kiva please make the same changes in that repo!

@mcstover @dyersituations this experiment was not implemented in `cms-page-server` repo

**Pull request merged**